storage/v1alpha1/frontend_virtio_blk.proto

```diff
@@ -66,6 +67,12 @@ message VirtioBlk {
     common.v1.ObjectKey volume_id = 3;
 
     int64 max_io_qps = 4;
+
```

Comment:
virtio-blk has a 1:1 mapping between the PCI endpoints and the block storage volumes. In other words, a virtio-blk controller only supports one namespace. That means the middle-end QoS is always sufficient and this isn't strictly needed. But I could be convinced to keep it for consistency.